torch/csrc/jit/script/tree_views.h

```diff
@@ -240,10 +242,13 @@ struct Expr : public TreeView {
       case TK_NE:
       case '+':
       case '-':
+      case TK_UNARY_MINUS:
       case '*':
+      case TK_STARRED:
       case '/':
       case TK_NOT:
       /* case '-': - unary minus */
+      /* case '*': - unary starred */
```

Comment:
These no longer have to be listed here. They were only mentioned because these tokens had two different meanings, which is no longer the case